packs/data/pathfinder-bestiary-3.db/kitsune-trickster.json

```
@@ -1650,15 +1652,15 @@
                     "value": ""
                 },
                 "rules": [],
-                "slug": null,
+                "slug": "change-shape",
                 "source": {
-                    "value": ""
+                    "value": "Pathfinder Bestiary"
                 },
                 "traits": {
-                    "custom": "",
+                    "custom": "[magical tradition]",
```

```suggestion
                    "custom": "",
```
hmm, I suspect this should be removed entirely since we don't have a way to do custom traits on items anymore with the new editor.